README.md

```diff
@@ -60,26 +62,29 @@ The `onChange` action is called with two arguments:
 - The model you just dragged
 
 ```js
-// app/routes/my-route.js
+// app/components/my-list/component.js
+
+export default class MyList extends Component {
+  @tracked lastDragged;
+  @tracked items = ['Coal Ila', 'Askaig', 'Bowmore'];
```

Comment:
Does this over simplify it, should I use object literals instead so can do `{{item.name}}` as in current examples?

Reply:
nah, I think this is good :100: